app/services/booking_response.rb

```diff
@@ -1,9 +1,44 @@
 class BookingResponse
-  def initialize(success:)
-    @success = success
+  ALREADY_PROCESSED_ERROR = 'already_processed'.freeze
+  PROCESS_REQUIRED_ERROR = 'process_required'.freeze
```

Comment:
I'll add a description on the intent of this class.

This class represents the result of the visit processing. Before the different states included the two above and were implicit. Now the response that staff can get includes extra states related to the result of the Nomis call (validation error / api error)